keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/EnvironmentSummary.kt

```diff
@@ -51,7 +51,8 @@ data class ArtifactVersions(
   val type: ArtifactType,
   val reference: String,
   val statuses: Set<ArtifactStatus>,
-  val versions: ArtifactVersionStatus
+  val versions: ArtifactVersionStatus,
+  val pinnedVersion: String?
```

Comment:
Added an optional field on the environment summary that will show you the pinned version, if it exists

Reply:
lovely